ui/components/VerifiedStatus.tsx

```diff
@@ -57,34 +35,30 @@ const VerifiedStatus = ({
   );
 };
 
-export const SourceIsVerifiedStatus: React.FC<{
-  automation?: Automation;
-  sourceRef?: ObjectRef;
-  source?: Source;
-}> = ({ automation, sourceRef, source }): JSX.Element => {
-  const isVerifiable = source
-    ? checkVerifiable({
-        name: source.name,
-        namespace: source.namespace,
-        clusterName: source.clusterName,
-        kind: source?.type,
-      })
-    : checkVerifiable(sourceRef || {});
-  if (!isVerifiable) return <Flex>-</Flex>;
+export const findVerificationCondition = (
+  a: VerifiableSource
+): Condition | undefined =>
+  a?.conditions?.find((condition) => condition.type === "SourceVerified");
 
-  if (source) return <VerifiedStatus source={source as VerifiableSource} />;
-  const {
-    name = "",
-    namespace = automation.namespace,
-    kind = "",
-    clusterName = automation.clusterName,
-  } = sourceRef || {};
-  const { data: verifiable } = useGetObject<VerifiableSource>(
-    name,
-    namespace,
-    Kind[kind],
-    clusterName
-  );
+export const VerifiedStatus = ({
+  source,
+}: {
+  source: VerifiableSource;
+}): JSX.Element | null => {
+  if (!source.isVerifiable) return null;
```

Comment:
Maybe this should be the `<Flex>-</Flex>` thing if we're using that now for empty cols